src/richie/apps/courses/models/course.py

```diff
@@ -346,16 +346,10 @@ class CoursePluginModel(PagePluginMixin, CMSPlugin):
         on_delete=models.CASCADE,
         related_name="course_plugins",
         limit_choices_to={
-            # There's a draft and a public course attached to the draft and
-            # the public parent. Without the first condition, 4 options are
-            # availables for each course. The second condition makes sure the
-            # parent is not a course. The third option filters out public
-            # course. The fourth option makes sure only courses show up.
-            # If there's is no parent, the course is filtered out.
-            "node__parent__cms_pages__publisher_is_draft": True,
-            "node__parent__cms_pages__course__isnull": True,
-            "publisher_is_draft": True,
-            "course__isnull": False,
+            "node__parent__cms_pages__publisher_is_draft": True,  # exclude course published parent
```

Comment:
Not really, it's more:
```suggestion
            "node__parent__cms_pages__publisher_is_draft": True,  # course has published parent
```

Reply:
done

Reply:
not done ;-)

Reply:
Ah I just bumped into a problem on my other PR that explains why we put this clause in the query. I think it's to avoid duplicate results that result when you join on `cms_pages` for the other clause that does it.

I'm now improving this situation on the other PR and I can propose a fixup here if you want?